specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-10-01/examples/TenantsGet.json

```diff
@@ -0,0 +1,17 @@
+{
+    "parameters": {
+      "api-version": "2019-01-01",
+      "billingProfileId": "dd21e577-ec81-45ec-b08a-622213737978",
+	    "billingAccountId": "0d49b9e9-dfe7-536e-ede4-bae1d63adbb7"
+    },
+    "responses": {
+      "200": {
+        "body": {
```

Comment:
the body payload should match the definition of TenantListResult

Reply:
its done now.